### DISCUSSION TITLE
about router.query

### DISCUSSION THREAD

**Discussion by aafnnp** [vercel/next.js#39092] [Help]
```
const { cardNo, billId, memberId } = router.query
console.log(cardNo, billId, memberId, 'query')
```
why this conole.log print three times?how to solve it?
![image](https://user-images.githubusercontent.com/12791216/181413547-7fb8189b-7cf1-4ac8-ba5c-5ce079d86424.png)

**Comment by GabenGar** [comment]
Because the component is re-rendered 3 times.
